ts_export/src/type_solver.rs

```diff
@@ -87,7 +97,7 @@ pub trait TypeSolver {
             SolverResult::Solved(inner_type, imports) => SolverResult::Solved(
                 TypeMember::PropertySignature(PropertySignature {
                     inner_type,
-                    name: PropertyName::Identifier(solver_info.field.attrs.name().serialize_name()),
+                    name: PropertyName::Identifier(solver_info.name.to_string()),
```

Comment:
I dont get why `to_string()` is required - isn't `MemberInfo::name` of type `String` ?

Reply:
it's `&String`, because we operate with `&MemberInfo` there